user_notification/models.py

```diff
@@ -24,22 +26,58 @@ class JobAlert(models.Model):
     def __str__(self):
         return str(self.user_account_id)
 
-    # Calculate the date of the next alert
     @classmethod
     def calc_check_date(cls, last_check_date, frequency_in_days):
```

Comment:
Change the function name to be more inductive

Reply:
fixed @bardielle  @guy9050